css/style.css

```diff
@@ -266,98 +282,176 @@ body {
+  font-weight: bold;
   font-size: 32px;
+  margin: 0;
+  letter-spacing: 0;
+  text-align: left;
   color: #172b4d;
-  max-width: 205px;
 }
 
-.tech {
+.project-info {
   display: flex;
-  justify-content: space-between;
-  max-width: 255px;
+  flex-direction: row;
   align-items: center;
-}
-
-.tech span:first-child {
-  color: #344563;
+  font-style: normal;
+  gap: 15px;
   font-weight: 600;
   font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
 }
 
-.tech span:nth-child(3) {
-  color: #7a869a;
-  font-weight: 600;
-  font-size: 13px;
+.project-info-title {
+  color: #344563;
+  margin: 0;
 }
 
-.tech span:nth-child(5) {
-  color: #7a869a;
-  font-weight: 600;
-  font-size: 13px;
+.project-info-item {
+  margin: 0;
 }
 
-.dot {
+.project-info img {
   width: 8px;
   height: 8px;
-  background-color: #c1c7d0;
-  border-radius: 50%;
+  margin: 0;
+  color: #c1c7d0;
 }
 
-.discription {
+.project-description {
+  width: 100%;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
   color: #344563;
-  font-weight: 400;
-  transition: 0.3s ease-in-out;
 }
 
-.discription:hover {
-  transform: translate(20px);
-}
-
-.lang {
+.tags {
   display: flex;
+  padding: 0;
   gap: 15px;
+  list-style: none;
+  align-items: center;
+  margin-top: 10px;
 }
 
-.lang .feild span {
+.tag-item {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  letter-spacing: 0.03em;
+  padding: 4px 12px;
   color: #6070ff;
-  background-color: #ebebff;
-  padding: 4px 12px 4px 12px;
+  background: #ebebff;
   border-radius: 8px;
-  font-size: 12px;
-  font-family: "poppins", sans-serif;
 }
 
-.feild-button .btn {
+.action-container {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  height: 72px;
+  margin-top: 12px;
+  padding: 0;
+}
+
+.action-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  font-style: normal;
   font-weight: 500;
-  padding: 12px;
   font-size: 17px;
-  color: #6070ff;
-  background-color: transparent;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  width: 124px;
+  height: 48px;
+  padding: 12px;
+  color: #396df2;
   border: 1px solid #6070ff;
+  background-color: #ffff;
   border-radius: 8px;
 }
 
-.feild-button .btn:hover {
-  background-color: #6070ff;
-  color: #ffff;
+/* Project Detail Modal Section */
+.modal-container {
+  display: none;
+  justify-content: space-around;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  max-height: 100vh;
+  background: #c1c7d0;
+  z-index: 1;
 }
 
-.feild-button .btn:active {
-  background-color: #2230d2;
-  color: #ffff;
+.works-modal {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  list-style: none;
+  width: 343px;
+  height: 713px;
+  padding: 18px 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
```

Comment:
- [OPTIONAL] Your menu hamburger is overlapping the `x` button of your popup in the mobile version making it impossible to close. Kindly review that.
![image](https://user-images.githubusercontent.com/61919617/173688411-79a4d4f0-2a53-4885-ad73-3c983cfacce3.png)
![image](https://user-images.githubusercontent.com/61919617/173688516-491c7de4-101b-4b08-a932-4fca3fd7e9e9.png)